dev/devicelab/lib/framework/utils.dart

```diff
@@ -478,9 +481,9 @@ List<String> _flutterCommandArgs(String command, List<String> options) {
         '5',
       ],
 
-    // DDS should be disabled for flutter drive in CI.
+    // DDS should generally be disabled for flutter drive in CI.
     // See https://github.com/flutter/flutter/issues/152684.
-    if (command == 'drive') '--no-dds',
+    if (command == 'drive' && !driveWithDds) '--no-dds',
```

Comment:
Need to pipe `driveWithDds` into this method though.

Reply:
I think the only spot it needs to go is where it's added in `perf_tests.dart` below.

Reply:
Oh derp. Reading fail.

Reply:
Fixed.